brage-import/src/main/java/no/sikt/nva/brage/migration/mapper/BrageNvaMapper.java

```diff
@@ -111,7 +112,7 @@ private static boolean isCristinRecord(Record record) {
     }
 
     private static List<String> filterOutEmptyValues(List<String> descriptions) {
-        return descriptions.stream().filter(string -> !string.isBlank()).toList();
+        return descriptions.stream().filter(StringUtils::isBlank).toList();
```

Comment:
Fikser forhåpentligvis bug:

	'java.lang.NullPointerException: Cannot invoke "String.isBlank()" because "string" is null at no.sikt.nva.brage.migration.mapper.BrageNvaMapper.lambda$filterOutEmptyValues$1(BrageNvaMapper.java:114) at java.base/java.util.stream.ReferencePipeline$2$1.accept(Unknown Source) at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(Unknown Source) at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source) at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source) at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source) at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(Unknown Source) at java.base/java.util.stream.ReferencePipeline.toArray(Unknown Source) at java.base/java.util.stream.ReferencePipeline.toArray(Unknown Source) at java.base/java.util.stream.ReferencePipeline.toList(Unknown Source) at no.sikt.nva.brage.migration.mapper.BrageNvaMapper.filterOutEmptyValues(BrageNvaMapper.java:114) at java.base/java.util.Optional.map(Unknown Source) at no.sikt.nva.brage.migration.mapper.BrageNvaMapper.extractAbstract(BrageNvaMapper.java:367) at no.sikt.nva.brage.migration.mapper.BrageNvaMapper.extractEntityDescription(BrageNvaMapper.java:211) at no.sikt.nva.brage.migration.mapper.BrageNvaMapper.toNvaPublication(BrageNvaMapper.java:72) at no.sikt.nva.brage.migration.lambda.BrageEntryEventConsumer.convertBrageRecordToNvaPublication(BrageEntryEventConsumer.java:367) at no.sikt.nva.brage.migration.lambda.BrageEntryEventConsumer.parseBrageRecord(BrageEntryEventConsumer.java:362) at no.sikt.nva.brage.migration.lambda.BrageEntryEventConsumer.lambda$handleRequest$0(BrageEntryEventConsumer.java:92) at nva.commons.core.attempt.Try.attempt(Try.java:33) at no.sikt.nva.brage.migration.lambda.BrageEntryEventConsumer.handleRequest(BrageEntryEventConsumer.java:92) at jdk.internal.reflect.GeneratedMethodAccessor110.invoke(Unknown Source) at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) at java.base/java.lang.reflect.Method.invoke(Unknown Source) at com.amazonaws.services.lambda.runtime.api.client.EventHandlerLoader$PojoMethodRequestHandler.handleRequest(EventHandlerLoader.java:285) at com.amazonaws.services.lambda.runtime.api.client.EventHandlerLoader$PojoHandlerAsStreamHandler.handleRequest(EventHandlerLoader.java:202) at com.amazonaws.services.lambda.runtime.api.client.EventHandlerLoader$2.call(EventHandlerLoader.java:905) at com.amazonaws.services.lambda.runtime.api.client.AWSLambda.startRuntime(AWSLambda.java:245) at com.amazonaws.services.lambda.runtime.api.client.AWSLambda.startRuntime(AWSLambda.java:197) at com.amazonaws.services.lambda.runtime.api.client.AWSLambda.main(AWSLambda.java:187)'